near-contract-standards/lib/non_fungible_token/utils.js

```diff
@@ -0,0 +1,42 @@
+import { near, assert } from "near-sdk-js";
```

Comment:
Let's add that trick for the lib folder so it's not shown in git changes.

Reply:
Done!